src/components/entity/ha-chart-line.html

```diff
@@ -0,0 +1,142 @@
+<link rel='import' href='../../../bower_components/polymer/polymer.html'>
+<link rel='import' href='ha-chart-base.html'>
+<dom-module id='ha-chart-line'>
+  <template>
+      <ha-chart-base publish data="[[chartData]]" selected-element ="{{selectedDataPoint}}"></ha-chart-base>
+  </template>
+</dom-module>
+<script>
+
+Polymer({
+  is: 'ha-chart-line',
+
+  useNativeCSSProperties: true,
```

Comment:
I thought that native was being used on supported platforms. Why toggle this? We shouldn't do this.